docs/DeveloperGuide.adoc

```diff
@@ -798,17 +797,56 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
 |`* * *` |user |add a new person |
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |user who wants to record notes |add a note entry for a person |record more information about the person
+
+|`* * *` |user interested in a contact's skills |get the technical skills of a candidate |to find suitable candidates
+
+|`* * *` |user who saves many resumes |open candidates' resumes from commandline |to quickly view their profiles
+
+|`* * *` |User |delete a tag from address book |to remove tags I am no longer interested in
+
+|`* *` |user |hide private contact details by default |minimize chance of someone else seeing them by accident
+
+|`* *` |user with multiple address books |export my contacts |import into another address book application
+
+|`* *` |user who views contacts |list frequently viewed candidates |to select the most probable candidates
+
+|`* *` |User with interest in recruitment |view candidates' LinkedIn profiles from commandline |to know about their work profiles
+
+|`* *` |User |view my search history |to keep track of my previous searches
+
+|`* *` |user interested in specific skills |search candidates by skill |to find suitable candidates easily
+
+|`* *` |user who is recruiting |search by graduation year |to find suitable full/part time candidates
+
+|`* *` |user interested in checking projects |access contacts' Github repositories |easily view and evaluate projects
+
+|`* *` |user who uses many applications |export search result to CSV/JSON |import them into other applications
```

Comment:
Is this different from "user with multiple address books | export my contacts | import into another address book application"?